eth-pm/package.json

```diff
@@ -17,9 +17,8 @@
     "uint8arrays": "^3.1.0"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "eject": "react-scripts eject",
+    "start": "GENERATE_SOURCEMAP=false PORT=3001 craco start",
```

Comment:
Oooh, does this remove all the sourcemap warnings? Interesting. I did do some upstream PR to help with those https://github.com/ChainSafe/js-libp2p-gossipsub/pull/337

Reply:
Yes, this doesn't generate sourcemap altogether; don't think we need it in our case.